packages/catalog-model/examples/apis/swapi-graphql.yaml

```diff
@@ -3,1174 +3,13 @@ kind: API
-      full "{ edges { node } }" version should be used instead.
-      """
-      films: [Film]
-    }
-
-    """An edge in a connection."""
-    type VehicleFilmsEdge {
-      """The item at the end of the edge"""
-      node: Film
-
-      """A cursor for use in pagination"""
-      cursor: String!
-    }
-
-    """A connection to a list of items."""
-    type VehiclePilotsConnection {
-      """Information to aid in pagination."""
-      pageInfo: PageInfo!
-
-      """A list of edges."""
-      edges: [VehiclePilotsEdge]
-
-      """
-      A count of the total number of objects in this connection, ignoring pagination.
-      This allows a client to fetch the first five objects by passing "5" as the
-      argument to "first", then fetch the total count so it could display "5 of 83",
-      for example.
-      """
-      totalCount: Int
-
-      """
-      A list of all of the objects returned in the connection. This is a convenience
-      field provided for quickly exploring the API; rather than querying for
-      "{ edges { node } }" when no edge data is needed, this field can be be used
-      instead. Note that when clients like Relay need to fetch the "cursor" field on
-      the edge to enable efficient pagination, this shortcut cannot be used, and the
-      full "{ edges { node } }" version should be used instead.
-      """
-      pilots: [Person]
-    }
-
-    """An edge in a connection."""
-    type VehiclePilotsEdge {
-      """The item at the end of the edge"""
-      node: Person
-
-      """A cursor for use in pagination"""
-      cursor: String!
-    }
-
-    """A connection to a list of items."""
-    type VehiclesConnection {
-      """Information to aid in pagination."""
-      pageInfo: PageInfo!
-
-      """A list of edges."""
-      edges: [VehiclesEdge]
-
-      """
-      A count of the total number of objects in this connection, ignoring pagination.
-      This allows a client to fetch the first five objects by passing "5" as the
-      argument to "first", then fetch the total count so it could display "5 of 83",
-      for example.
-      """
-      totalCount: Int
-
-      """
-      A list of all of the objects returned in the connection. This is a convenience
-      field provided for quickly exploring the API; rather than querying for
-      "{ edges { node } }" when no edge data is needed, this field can be be used
-      instead. Note that when clients like Relay need to fetch the "cursor" field on
-      the edge to enable efficient pagination, this shortcut cannot be used, and the
-      full "{ edges { node } }" version should be used instead.
-      """
-      vehicles: [Vehicle]
-    }
-
-    """An edge in a connection."""
-    type VehiclesEdge {
-      """The item at the end of the edge"""
-      node: Vehicle
-
-      """A cursor for use in pagination"""
-      cursor: String!
-    }
+  definition:
+    $text: https://github.com/graphql/swapi-graphql/blob/master/schema.graphql
```

Comment:
Hm, this is an unrelated change - drawback being, now you're subject to github rate limits instead of all data being local.

I'm actually OK with it though. Should work fine as long as we have the implicit github.com integration always being added whether it's in one's config or not.

Reply:
Good point. I'll revert this change.

Reply:
>Hm, this is an unrelated change - drawback being, now you're subject to github rate limits instead of all data being local.

In the end we are already doing it for most other API files, but I would prefer to keep this as-is to have another test case without `$text`.

Reply:
Ah of course, you are right. But OK, revert sounds fine anyway

Reply:
Sorry already reverted. Happy to change it back in another PR for another example.